tests/filterparser/test_filterparser.py

```diff
@@ -186,6 +186,7 @@ def test_list_properties(self):
 
         # OPTIONAL:
         self.assertIsInstance(self.parse('elements HAS ONLY "H","He","Ga","Ta"'), Tree)
+        self.assertIsInstance(self.parse('elements HAS ALL "H","He","Ga","Ta"'), Tree)
```

Comment:
Also, `HAS ANY` should be added?

Reply:
Yep, that's next, got distracted by the mongomock issues but will get back on this